AirportPanel/Program.cs

```diff
@@ -0,0 +1,490 @@
+            s += $"{(TerminalName)Enum.GetNames(typeof(TerminalName)).Length})";
+            Console.WriteLine($"| {s}" + new string(' ', 56 - s.Length) + " |");
+            Console.WriteLine("| Input gate in the terminal (1 to 999)                    |");
+            Console.WriteLine("| Input flight status (help at the bottom of the screen)   |");
+            Console.ForegroundColor = ConsoleColor.White;
+
+            Flight flight = new Flight
+            {
+
+                InOut = InputInOut(61, 2),
+                Number = InputNumberGate(61, 3),
+                DTFlight = InputDT(61, 4),
+                City = InputCity(61, 5),
+                Airline = InputAirline(61, 6),
+                Terminal = InputTerminal(61, 7),
+                Gate = InputNumberGate(61, 8),
+                Status = InputStatus(61, 9),
+            };
+
+            Console.BackgroundColor = ConsoleColor.Black;
+            Console.SetCursorPosition(0, 40);
+            Console.Write(new string(' ', 164));
+            Console.SetCursorPosition(0, 40);
+            Console.WriteLine("MENU: 1-ViewList 2-AddFlight 3-DeliteFlight 4-EditFlight 5-FindFlight 6-SpecialFunction Q-Quit");
+
+            return flight;
+        }
+        public static void TableHead()
+        {
+            string s = null;
+
+            Console.ForegroundColor = ConsoleColor.White;
+            Console.BackgroundColor = ConsoleColor.Red;
+            Console.WriteLine(CentrStr("Information panel", 164));
+            Console.BackgroundColor = ConsoleColor.Blue;
+            Console.WriteLine(new string('-', 164));
+            s += "|" + CentrStr("In", 80);
+            s += "||" + CentrStr("Out", 80) + "|";
+            Console.WriteLine(s);
+            Console.WriteLine(new string('-', 164));
+            s = "|" + CentrStr("№",5);
+            s += "|" + CentrStr("Date/time", 18);
+            s += "|" + CentrStr("City", 9);
+            s += "|" + CentrStr("Airline", 17);
+            s += "|" + CentrStr("Terminal", 8);
+            s += "|" + CentrStr("Gate", 5);
+            s += "|" + CentrStr("Status", 12) + "|";
+            s = s + s;
+            Console.WriteLine(s);
+            Console.WriteLine(new string('-', 164));
+        }
+        public static string CentrStr(string s,int l)
+        {
+            string result;
+            if ((l - s.Length) % 2 == 0)
+            {
+                result = new string(' ', (l - s.Length) / 2) + s + new string(' ', (l - s.Length) / 2);
+            }
+            else
+            {
+                result = new string(' ', (l - s.Length) / 2) + s + new string(' ', (l - s.Length) / 2 + 1);
+            }
+            return (result);
+        }
+        public static bool InputInOut(int left, int top)
+        {
+            Console.SetCursorPosition(0, 40);
+            Console.Write(new string(' ', 164));
+            Console.SetCursorPosition(left, top);
+            int i = -1;
+            string s = null;
```

Comment:
Use more informative variable name

Reply:
agree, "c" and "l" needed to be described in more detail, for example, Text and CellLength